libutils/json.c

```diff
@@ -76,6 +76,20 @@ struct JsonElement_
 // JsonElement Functions
 // *******************************************************************************************
 
+const char *JsonContainerTypeToString(const JsonContainerType type)
```

Comment:
I'd make this `static inline` and moved it to the header, but not a biggie.

Reply:
again, doing it as-it-was-already with other *TypeToString() functions.

Reply:
Looking it would seem there are MANY functions which could be moved to the header and made `static inline`. Not sure where/if we should start/stop. @vpodzime lemme know what you think... just look through the first part of the file. e.g.

```
127	const char *JsonElementGetPropertyName(const JsonElement *const element)
128	{
129	    assert(element != NULL);
130	
131	    return element->propertyName;
132	}
```